src/python_testing/matter_testing_infrastructure/idt_tools.gni

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2025 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+
+declare_args() {
+  # To build the chip-testing Python wheel with a full set of files
+  # for the Interoperability Debug Tool (IDT), set the following flag to `true`.
+  include_idt_files = false
+}
+
+idt_files = [
+  "${chip_root}/credentials/fetch_paa_certs_from_dcl.py",
+  "${chip_root}/src/python_testing/post_certification_tests/production_device_checks.py",
+  "${chip_root}/src/python_testing/TC_DA_1_2.py",
```

Comment:
Why test cases are baked into the testing infrastructure package? For now in the testing infrastructure we had tools/helpers required to write tests. Since now this package will contain test cases itself, why not all tests? Maybe it would be better to create a separate Python package like `matter-post-certification-tests` which would depend on `matter-testing`?

Reply:
I don't feel that we need to create a separate goal/package for including IDT tools, why not reuse the existing one and put it under the flag that defaults to false? All tests are not needed in IDT (https://github.com/CHIP-Specifications/matter-interoperability), we only want to export what is necessary, tests themselves could be considered as tools reused in IDT.

Reply:
I proposed an alternative solution in the PR below https://github.com/project-chip/connectedhomeip/pull/39748
Please let me know your thoughts.